services/base-jdbc/src/main/java/org/eclipse/hono/service/base/jdbc/config/JdbcDeviceProperties.java

```diff
@@ -17,14 +17,13 @@
 import com.fasterxml.jackson.annotation.JsonInclude.Include;
 
 /**
- *
+ * JDBC properties for the device store.
```

Comment:
FMPOV it  would be less misleading if we could rename this class to *JdbcDeviceStoreProperties* ...